Rules/CTF/Scripts/CTF_Banners.as

```diff
@@ -0,0 +1,135 @@
+#define CLIENT_ONLY;
+
+#include "GameStateBanners.as";
+
+Banner@ getBuildBanner()
+{
+	u32 duration = 5 * getTicksASecond();
```

Comment:
move seconds to const variable at top of file (for others as well) and do `seconds * getTicksASecond()`

Reply:
resolved